src/libs/my-dogs/ui/my-dogs-ui/my-dogs-ui.component.ts

```diff
@@ -39,7 +37,10 @@ export class MyDogsUiComponent implements OnInit {
 
   constructor(private readonly _fb: NonNullableFormBuilder) {}
 
-  ngOnInit(): void {}
+  saveDog(): void {
+    const dog: Omit<Dog,'id'> = {breed: this.form.value.breed!, name: this.form.value.name!};
```

Comment:
wystarczyłoby this.form.getRawValue() zamiast {breed: this.form.value.breed!, name: this.form.value.name!}

Reply:
musisz strzelać z "!" bo jeśli kotrolka breed była by disabled to jej wartość byłaby nullem, taka ciekawostka, getRawValue jest lepszym pomysłem w większości przypadków